web/settings/modulconfiglp.php

```diff
@@ -2354,6 +2408,10 @@ function display_socmodul() {
 								showSection('#socoldevccwarning');
 								showSection('#socmid');
 							}
+							if($('#socmodul').val() == 'soc_vwid') {
+								showSection('#socvwidlpxinfo');
```

Comment:
Den Bereich finde ich nirgends?
```suggestion
```

Reply:
Hier der fehlende Teil.
`
								<div id="socvwidlpxinfo" class="mt-1 alert alert-danger hide">
									New VW ID SOC module based on python3 and libwvid.py.
								</div>
`
Kannst Du das einbauen oder soll ich ein neues commit machen?

Reply:
Mach Du das bitte. Es müsste dann aber für jeden LP ein Block rein.